libs/wire-subsystems/test/unit/Wire/ScimSubsystem/InterpreterSpec.hs

```diff
@@ -42,13 +42,25 @@ type AllDependencies =
 
 runDependencies ::
   forall a.
+  (HasCallStack) =>
   [User] ->
   Map TeamId [TeamMember] ->
   Sem AllDependencies a ->
   Either ScimSubsystemError a
 runDependencies initialUsers initialTeams =
+  either (error . show) id . runDependenciesSafe initialUsers initialTeams
+
+runDependenciesSafe ::
+  forall a.
+  (HasCallStack) =>
+  [User] ->
+  Map TeamId [TeamMember] ->
+  Sem AllDependencies a ->
+  Either UGS.UserGroupSubsystemError (Either ScimSubsystemError a)
```

Comment:
please don't involve user group subsystem here, this is supposed to work on spar, so the interpreter should use brig internal api instead.

the errors returned from there should be translated into errors from the hscim library wrapped in ScimSubsystemError.

(i want to remove ScimSubsystemError and replace it with ScimError, but in a different PR.)